docs/k8s/getting-started-kic.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Getting Started - Kubernetes Ingress Controller
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+import K8sIngressInstallExample from "/examples/k8s/install-ingress.mdx";
+import K8sGatewayAPIInstallExample from "/examples/k8s/install-gateway-api.mdx";
+
+import K8sIngressModifyHeadersExample from "/examples/k8s/http-request-headers.mdx";
+import K8sGatewayModifyHeadersExample from "/examples/k8s/http-request-headers-gateway.mdx";
+
+import K8sIngressRedirectExample from "/examples/k8s/http-request-redirect.mdx";
+import K8sGatewayRedirectExample from "/examples/k8s/http-request-redirect-gateway.mdx";
+
+import StaticDomainKubernetesExample from "/examples/k8s/http-static-domain.mdx";
+import StaticDomainKubernetesGatewayExample from "/examples/k8s/http-static-domain-gateway.mdx";
+
+import StaticDomainKubernetesPathExample from "/examples/k8s/http-static-domain-path.mdx";
+import StaticDomainKubernetesGatewayPathExample from "/examples/k8s/http-static-domain-gateway-path.mdx";
+
+# Kubernetes Ingress Controller
+
+## Overview
+
+The Kubernetes Ingress Controller within the Kubernetes Operator allows access to k8s Services using Kubernetes [Ingress CRDs](https://kubernetes.io/docs/concepts/services-networking/ingress/). More details on how these are derived can be found [here](/docs/k8s/with-edges).
```

Comment:
Let's change the link text from `here` to something more meaningful

Reply:
Switched this to something more clear.